docs/getting-started/data-masking.md

```diff
@@ -0,0 +1,377 @@
+# Data Masking
+
+Data masking is a critical security feature that allows you to protect sensitive information while maintaining data utility for development, testing, and analytics purposes. ingestr provides comprehensive masking capabilities that can be applied to any column during the ingestion process.
+
+## Overview
+
+Data masking transforms sensitive data into a protected format while preserving the structure and type of the original data. This is essential for:
+
+- **Compliance** with regulations like GDPR, CCPA, HIPAA
+- **Security** in development and testing environments
+- **Privacy** protection in analytics and reporting
+- **Data sharing** with third parties or external systems
```

Comment:
Hi. Thanks for the excellent patch. It might make sense to also add a bit of name dropping about [PII](https://en.wikipedia.org/wiki/Personal_data) scrubbing here, because I think it fits into the topic well.